DifferentiationInterface/Project.toml

```
@@ -38,7 +38,7 @@ DifferentiationInterfaceFiniteDiffExt = "FiniteDiff"
 DifferentiationInterfaceFiniteDifferencesExt = "FiniteDifferences"
 DifferentiationInterfaceForwardDiffExt = ["ForwardDiff", "DiffResults"]
 DifferentiationInterfaceGTPSAExt = "GTPSA"
-DifferentiationInterfaceMooncakeExt = "Mooncake"
+DifferentiationInterfaceMooncakeExt = ["ChainRulesCore", "Mooncake"]
```

```suggestion
DifferentiationInterfaceMooncakeExt = "Mooncake"
```
I think you're no longer piggybacking on ChainRules? Besides, even if you were, this addition would mean that users need to load ChainRulesCore in order to call Mooncake through DI, effectively breaking every downstream code that relies on `AutoMooncake`. Does that make sense?

---

yup